tests/endtoend/http_functions/http_functions_stein/function_app.py

```diff
@@ -1,7 +1,9 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+from datetime import datetime
 import logging

```

Comment:
I don't think logging is needed. Can be removed.

Reply:
we are using logging in default_template

Reply:
Oh sorry,  didn't see the default_template function